Tribler/Core/Session.py

```diff
@@ -420,15 +419,16 @@ def start(self):
         def after_upgrade(_):
             self.upgrader = None
 
+
```

Comment:
Why was this newline added here? Methods require one newline between them right?

Reply:
Just a mistype. I'm so used to `pre-commit` on `devel` that I don't consider formatting at all ))

Reply:
@devos50 , I love how you care about formatting even on a branch that is not ever going to be merged in mainline :wink:

Reply:
Line deleted.
Squash and merge at will.